platform-hub-api/app/controllers/kubernetes/groups_controller.rb

```diff
@@ -1,10 +1,95 @@
 class Kubernetes::GroupsController < ApiJsonController
 
+  before_action :find_group, only: [ :show, :update, :destroy ]
+
+  skip_authorization_check only: [ :index, :show ]
+  authorize_resource class: KubernetesGroup, :except => [ :index, :show ]
+
+  # GET /kubernetes/groups
+  def index
+    groups = KubernetesGroup.order(:name)
+    render json: groups
+  end
+
+  # GET /kubernetes/groups/:id
+  def show
+    render json: @group
+  end
+
+  # POST /kubernetes/groups
+  def create
+    group = KubernetesGroup.new(group_params)
+
+    if group.save
```

Comment:
I think in all the controllers so far we've been doing this so that we can return back the errors in the `else` block. Unless I'm missing something specific to this case?